commands/Purge300

```diff
@@ -0,0 +1,94 @@
+    {
```

Comment:
This command type is not compatible with this bot. you will need to rewrite this command to typescript for it to work